package.json

```
@@ -1,11 +1,16 @@
 {
   "name": "svelte-portal",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

```suggestion
  "version": "2.2.0",
```
Version will be bumped automatically